cardinal-components-block/src/main/java/org/ladysnake/cca/internal/block/StaticBlockComponentPlugin.java

```diff
@@ -41,13 +41,7 @@
 import org.ladysnake.cca.internal.base.asm.StaticComponentLoadingException;
 import org.ladysnake.cca.internal.base.asm.StaticComponentPluginBase;
 
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.LinkedHashMap;
-import java.util.LinkedHashSet;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
```

Comment:
I'm partial to standardize on avoiding star imports, could you configure your formatter to avoid them in this codebase ?